src/main/java/Duke/DukeException/DukeArrayException.java

```diff
@@ -0,0 +1,6 @@
+package main.java.Duke.DukeException;
+
+public class DukeArrayException extends DukeException {
```

Comment:
Minor issue, but this file can be removed if it is now obsolete.

Reply:
To rephrase, it would be better to group this exception with your DukeException class below.